docs/manual-EN/3.build-develop-and-administration/3.configurations/0.system-requirement.md

```diff
@@ -54,7 +54,7 @@ Take AWS EC2 c5d.xlarge as an example:
 ## Resource Estimation (Three Replicas)
 
 * Storage space (full cluster): number of edges and vertices * average bytes of attributes * 6
-* Memory (full cluster): number of edges and vertices * 15 bytes + number of RocksDB instances * (write_buffer_size * max_write_buffer_number + rocksdb_block_cache), where each directory in the `--data_path` item in the `etc/nebula-storaged.conf` file corresponds to a RocksDB instance
+* Memory (full cluster): number of edges and vertices * 15 bytes + number of RocksDB instances * (write_buffer_size * max_write_buffer_number + rocksdb_block_cache), where each directory in the `--data_path` item in the `etc/nebula-storaged.conf` file corresponds to a RocksDB instance. You can decrease the memory of bloomfitler by setting the `enable_partitioned_index_filter` parameter to `true`.q
```

Comment:
the memory of bloomfitler ==> the memory size of bloomfitler?

delete the "q" at the end.